### DISCUSSION TITLE
Helps : React Update Failure

### DISCUSSION THREAD

**Discussion by blackstarsama** [vercel/community#1592] [Help]
### Summary

I deployed my portfolio in Reactjs but when I try to make changes and publish them I get this error.

### Additional information

<html>
<body>

npm ERR! code EINVALIDTAGNAME
--
02:21:03.003 | npm ERR! Invalid tag name ">=^16.0.0" of package "react@>=^16.0.0": Tags may not have any characters that encodeURIComponent encodes.

</body>
</html>

**Comment by swarnava** [comment]
Can you double check your package.json? Sounds like there is a missing `"` somewhere

**Reply by blackstarsama** [reply]
I checked and everything is ok with the "normally.  

Please do you have another lead?

**Reply by swarnava** [reply]
Can you share a minimal reproducible git repo which we can take a look?

**Reply by blackstarsama** [reply]
https://github.com/blackstarsama/Portfolio

**Reply by swarnava** [reply]
Did you follow a guide, such as [Using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel)?

**Reply by blackstarsama** [reply]
i'm going to check it! thanks

i fix my problem with " npm install --legacy-peer-deps " in the Build & Development Settings of vercel, but it's as if I bypassed the problem without fixing it
